README.md

```diff
@@ -34,241 +30,10 @@ If you already cloned and just need to get the submodules, you can run:
-
-- Translation: WMT'14 EN-DE, WMT'17 EN-RU. Scripts to prepare the WMT data are in [`scripts/wmt/`](scripts/wmt/).
-- Language modeling: [Billion Word Benchmark](http://www.statmt.org/lm-benchmark/), [WikiText103](https://einstein.ai/research/the-wikitext-long-term-dependency-language-modeling-dataset). We use the English sentence tokenizer from [NLTK toolkit](https://www.nltk.org/) [Punkt Tokenizer Models](http://www.nltk.org/nltk_data/) to preprocess WikiText103 corpus. Note that it's only used in breaking paragraphs into sentences. It will use default tokenizer on word level as all other tasks unless otherwise specified. We don't do any preprocessing on BWB corpus.  
-- Image captioning: MSCOCO Dataset (http://cocodataset.org/#download). Specifically we use the following splits: 2017 Train images [118K/18GB], 2017 Val images [5K/1GB], 2017 Train/Val annotations [241MB].
-- Reddit: [reddit_comments dataset](https://bigquery.cloud.google.com/dataset/fh-bigquery:reddit_comments). Specifically we use the 2008 and 2009 tables.
-- DisSent: Details for preparing the corpora are in [`scripts/dissent/README`](scripts/dissent/README).
-- DNC (**D**iverse **N**atural Language Inference **C**ollection), i.e. recast data: The DNC is available [online](https://github.com/decompositional-semantics-initiative/DNC). Follow the instructions described there to download the DNC.
-- CCG: Details for preparing the corpora are in [`scripts/ccg/README`](scripts/ccg/README).
-- Edge probing analysis tasks: see _Papers_ below or [`probing/data`](probing/data/README.md) for more information.
-
-To incorporate the above data, placed the data in the data directory in its own directory (see task-directory relations in `src/preprocess.py` and `src/tasks.py`.
-
-### Running
-
-To run an experiment, make a config file similar to `config/demo.conf` with your model configuration. You can use the `--overrides` flag to override specific variables. For example:
-```sh
-python main.py --config_file config/demo.conf \
-    --overrides "exp_name = my_exp, run_name = foobar, d_hid = 256"
-```
-will run the demo config, but output to `$JIANT_PROJECT_PREFIX/my_exp/foobar`.
-
-To run the demo config, you will have to set environment variables. The best way to achieve that is to follow the instructions in [user_config_template.sh](user_config_template.sh)
-*  $JIANT_PROJECT_PREFIX: the where the outputs will be saved.
-*  $JIANT_DATA_DIR: location of the saved data. This is usually the location of the GLUE data in a simple default setup.
-*  $WORD_EMBS_FILE: location of any word embeddings you want to use (not necessary when using ELMo, GPT, or BERT). You can download GloVe (840B) [here](http://nlp.stanford.edu/data/glove.840B.300d.zip) or fastText (2M) [here](https://s3-us-west-1.amazonaws.com/fasttext-vectors/crawl-300d-2M.vec.zip).
-
-To have `user_config.sh` run automatically, follow instructions in [scripts/export_from_bash.sh](export_from_bash.sh). 
-
-## Command-Line Options
```

Comment:
Where does this go? Be careful not to delete any documentation...

Reply:
Again, this is in https://jiant.info/documentation#/?id=command-line-options